eva/catalog/catalog_manager.py

```diff
@@ -81,143 +85,265 @@ def _shutdown_catalog(self):
         logger.info("Shutting catalog")
         drop_db()
 
-    def create_metadata(
+    "Table catalog services"
+
+    def insert_table_catalog_entry(
```

Comment:
Diff meaning between `create` and `insert`?

Reply:
I feel since we are adding an entry into the catalog table, naming it `insert` is better than `create`.

Reply:
Sure. Just want to make sure we follow the same convention for all catalog APIs.